src/ParseObject.js

```diff
@@ -34,10 +33,12 @@ import {
 import ParsePromise from './ParsePromise';
 import ParseQuery from './ParseQuery';
 import ParseRelation from './ParseRelation';
+import * as SingleInstanceState from './SingleInstanceState';
```

Comment:
I hope you won't hate me for this. But this should be `SingleInstanceStateController`. I feel strongly about this since `SingleInstanceState` sounds like model, while `SingleInstanceStateController` clearly... a controller.